contrib/vagrant/provision-util.sh

```diff
@@ -317,7 +317,7 @@ os::provision::start-node-service() {
   fi
 
   cmd="/usr/bin/openshift start node --loglevel=${LOG_LEVEL} \
---config=$(os::provision::get-node-config ${config_root} ${node_name})"
+--config="$(os::provision::get-node-config ${config_root} ${node_name})""
```

Comment:
This won't work like you are intending it to. http://mywiki.wooledge.org/BashFAQ/050